test-clients/src/main/java/com/hedera/services/bdd/suites/perf/topic/SubmitMessageLoadTest.java

```diff
@@ -148,7 +146,6 @@ private static HapiApiSpec runSubmitMessages() {
 								.payingWith(GENESIS)
 								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "4000",
 										"hapi.throttling.ops.consensusSubmitMessage.capacityRequired", "1.0")),
-						reduceFeeFor(ConsensusSubmitMessage, 2L, 3L, 3L),
```

Comment:
Why is this line removed? Will this not cause any issues in perf tests ?

Reply:
See the comment above.